Stabilize broadcasting of incremental command updates

ndoschek opened the pull request:
Fixes #66 

Signed-off-by: Nina Doschek <ndoschek@eclipsesource.com>

martin-fleck-at reviewed (changes requested):
Hi Nina, thank you very much for your change. I had a look at your changes and ran into a few issues:
- Test cases no longer compile, specifically ModelControllerTest and SessionControllerTest. Please ensure that they compile and run through.
- I seem to get updates multiple times: You can test this with the default client in the menu SuperBrewer3000.json > subscribeWithTimeout and then performing an edit operation. It may not happen the first time but if you wait for the timeout and redo those steps you should see multiple updates. You can do the same with SuperBrewer3000.coffee and its subscriptions. Could you look into that?

ndoschek commented:
Thank you for your review Martin and spotting my sloppy mistakes!
I fixed the test cases and they run through on my setup most of the time, occasional failures are known and caused by #63.
The multiple updates were caused due to a issue in [@eclipse-emfcloud/emfcloud-modelserver-theia](https://github.com/eclipse-emfcloud/emfcloud-modelserver-theia), i fixed it along with the current PR https://github.com/eclipse-emfcloud/emfcloud-modelserver-theia/pull/28.

martin-fleck-at reviewed (approved):
Thank you for your update, everything works as expected now! :+1:
